styles/all/template/js/collapsiblecategories.js

```diff
@@ -0,0 +1,58 @@
+(function ($) {  // Avoid conflicts with other libraries
+
+	'use strict';
+
+	$(function () {
+		$('a.category[data-id] + .forabg').each(function () {
+			var $this = $(this),
+				id = $this.prev().attr('data-id'),
+				hidden = $this.prev().attr('data-hidden'),
+				$header = $this.find('li.header'),
+				$content = $this.find('.topiclist.forums');
+
+			if (!$header.length || !$content.length) {
+				return;
+			}
+
+			// Add button
+			$header.append('<a class="collapse-btn collapse-' + ((hidden) ? 'show' : 'hide') + '" data-id="' + id + '" data-ajax="phpbb_collapse"></a>');
```

Comment:
As I have thought about this now, why root_path? Can whe expect that controller routes are rewritten? And why without sid?
Or was it just a suggestion that we have to do something there?

I mean I would have done 
```php
$this->helper->route('phpbb_collapsible');
```